deps/verifier/src/tdx/quote.rs

```diff
@@ -563,5 +619,10 @@ mod tests {
         let quote_bin = fs::read(quote).unwrap();
         let res = ecdsa_quote_verification(quote_bin.as_slice()).await;
         assert!(res.is_ok(), "{res:?}");
+
+        let claims = serde_json::to_string_pretty(&res.unwrap())
+            .expect("Custom claims are available.");
+
+        let _ = fs::write(format!("{quote}.json"), claims);
```

Comment:
We might need to add the file to `.gitignore`?

Reply:
I don't see where the output is used so I'd suggest we just drop the write.

Reply:
I have replaced it with expected comparison - should do the trick